python_app/app.py

```
@@ -0,0 +1,64 @@
+    }
+    return jsonify(result)
+
+
+@app.route("/api/connection-state")
+def api_connection_state() -> Any:
+    state: Any = get_mock_connection_state()
+    return jsonify({"state": state.value})
+
+
+@app.route("/api/parse", methods=["POST"])
+def api_parse() -> Any:
```

Ruff (0.15.6)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `api_parse`

(ANN401)